### PR TITLE
Project hello-rails-back-end: Rails api

### PR DESCRIPTION
**Pull request by sarazaToussaint**
### In this Pull request I: 

- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize the project with Git.
- Use Postgres as a database.
- Create a table for storing the messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from the table.
- Include the link for the Front-end app in the README.md file

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```
Comment (mikethreels) [inline config/routes.rb]: - [x] a route should have been created here.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Toussaint, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- followed the instructions very carefully regarding creating the PR.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (peterrobert) [reply config/routes.rb]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @sarazaToussaint, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (sarazaToussaint) [reply config/routes.rb]: Hi @mikethreels, thank you for your review

**Comment by sarazaToussaint** [comment]
Dear @peterrobert, thank you for approving my PR.

**Pull request merged**